packages/app-portal/src/systems/Ecosystem/data/projects.json

```
@@ -89,14 +89,36 @@
   },
   {
     "isLive": false,
-    "name": "Elix Finance",
-    "url": "https://elix.finance/",
-    "tags": ["DeFi"],
-    "description": "Elix Finance is a hybrid automated market maker featuring automated concentrated liquidity.",
-    "github": "https://github.com/elixfinance",
-    "twitter": "https://twitter.com/elixfinance",
-    "discord": "https://discord.gg/avrdk2tAY6",
-    "image": "elix"
+    "name": "Connext",
+    "url": "https://www.connext.network/",
```

(nit) just to use the same displaying pattern

```suggestion
    "url": "https://connext.network/",
```

if you don't mind, I realized there are only other 9 using `www.` 

Could you please remove them?

---

Yep will do